content/en/agent/faq/linux-key-rotation-2024.md

```diff
@@ -0,0 +1,201 @@
+---
+title: 2024 Linux Key Rotation
+kind: guide
+private: true
+aliases:
+  - /agent/guide/linux-key-rotation-2024
+---
+
+As a common best practice, Datadog periodically rotates the keys and certificates used to sign Datadog's Agent packages. The Agent packages includes:
+
+- the different flavours of agent (datadog-agent, datadog-iot-agent, datadog-heroku-agent, datadog-dogstatsd, datadog-fips-proxy and datadog-agent-dbg).
```

Comment:
```suggestion
- the different flavours of Agent (`datadog-agent`, `datadog-iot-agent`, `datadog-heroku-agent` and `datadog-dogstatsd`).
```

FIPS proxy is not an Agent flavor but a separate package. The dbg packages are very rarely used, let's not mention them to prevent confusion.